src/ch/blinkenlights/android/vanilla/SeekBarPreference.java

```diff
@@ -103,7 +106,17 @@ protected View onCreateDialogView()
 
 		SeekBar seekBar = (SeekBar)view.findViewById(R.id.seek_bar);
 
-		int maxValue = (PrefKeys.SHAKE_THRESHOLD.equals(getKey()) ? 300 : 150);
+		int maxValue;
+		switch (getKey()) {
+			case PrefKeys.SHAKE_THRESHOLD:
```

Comment:
Sure, no problem. 

I added 

```
android {
...
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_5
        targetCompatibility JavaVersion.VERSION_1_5
    }
...
}
```
to the `build.gradle`. Apparently Java 5 doesn't allow `@Override` for interface methods, this was added in Java 6. It's not a compilation error, but it does mean that developers can't rely on reported `@Override` errors being consistently errors. If the project's set for 1.5, I think it would be wise to remove them.